packages/sc-deployer/src/index.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-len */
```

Comment:
we should avoid adding exception

`npm run fmt` to format the code

```suggestion
```

Reply:
I have formatted the code now and remove eslint overrides